### ISSUE TITLE
stop generating full bloomfilters for each blocklist change (drop non-stash support)

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#15426]
For each change of the blocklist (blocked added, changed, or deleted) we currently generate stash JSON, _and_ a new blocklist.  Stash support is enabled by a Firefox preference but has been enabled by default since the v3 blocklist was launched in Firefox 78 (including the ESR).

Since there are no release versions of Firefox relying on the non-stash v3 blocklist we should remove the support to avoid both the unnecessary complication/execution and the additional storage on kinto/remote-settings.

**Comment by AlexandraMoga** [comment]
@eviljeff I've blocked two new guids, changed the version range for some existing blocks and deleted a block, then I verified that these changes were reflected in the new stash on https://kinto.dev.mozaws.net/v1/buckets/blocklists_addons_dev_allizom_org/collections/addons-bloomfilters/records

The `bloomfilter_full` field is no longer present in the response, only the stash data:


```
{
    "data": [
        {
            "stash": {
                "blocked": [
                    "{0ec56aba-6955-43fb-a5cf-ed3f3ab66e7e}:3.2.7webext",
                    "{0ec56aba-6955-43fb-a5cf-ed3f3ab66e7e}:3.2.8webext",
                    "{0ec56aba-6955-43fb-a5cf-ed3f3ab66e7e}:3.3webext",
                    "{7a36e9c2-c41d-4279-ae15-f1cf59c5f501}:1.2",
                    "{7a36e9c2-c41d-4279-ae15-f1cf59c5f501}:1.0",
                    "{0ec56aba-6955-43fb-a5cf-ed3f3ab66e7e}:3.2.9webext"
                ],
                "unblocked": [
                    "{2e9df256-208a-4e7f-b795-bcaca394d9f1}:1.23.8",
                    "{4b05a4f0-c0ca-4041-970e-c7352fbb2037}:1.12"
                ]
            },
            "key_format": "{guid}:{version}",
            "stash_time": 1600778107452,
            "id": "0948db75-1860-4dd8-9f9a-d0a50d9cea8d",
            "last_modified": 1600778112936
        }
    ]
}
```

Let me know if there is anything else I should be testing besides what I've already covered.
